tsconfig.json

```diff
@@ -1,17 +1,13 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "esModuleInterop": true,
     "inlineSources": true,
     "lib": ["ES2020"],
     "module": "CommonJS",
-    "moduleResolution": "Node",
-    "outDir": "dist",
-    "rootDir": "src",
+    "moduleResolution": "node",
+    "noEmit": true,
     "sourceMap": true,
     "strict": true,
-    "target": "ES2017"
-  },
-  "exclude": ["./src/**/*.test.ts"],
```

Comment:
I just tried building using this config and saw it fail due to an invalid type in `dist` from when I was working on another branch. Maybe we should exclude `dist` here? Or include both the real source and test modules?

Reply:
I'm not sure I understand. Did you run `yarn build` then switch to another branch? Or did you switch to another branch and then run `yarn build`? Or were you using this in another project?

Reply:
I was on `main`, testing whether changing the signature of `_start` in the base block tracker would introduce a type error if the subclass wasn't changed as well. It did, there was an error upon building. I discarded these changes and switched back to this branch, and upon running `yarn build`, TypeScript (not having any `include` or `exclude` rules to guide it) looked for _all_ files/types in the current directory and tried to compile them. This included the `.d.ts` files in the `dist` directory that included that build error I introduced.

Reply:
I'm not getting this issue. I think it should be fixed, now that I've updated `yarn build` to use `tsconfig.build.json`.

Reply:
My repro did indeed rely upon using the non-build tsconfig. Could repro now by running `tsc` directly rather than via `yarn build`.